packages/server-core/src/user/strategies/apple.ts

```diff
@@ -0,0 +1,165 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import multiLogger from '@etherealengine/common/src/logger'
+import { avatarPath, AvatarType } from '@etherealengine/common/src/schemas/user/avatar.schema'
+import { AuthenticationRequest, AuthenticationResult } from '@feathersjs/authentication'
+import { Paginated, Params } from '@feathersjs/feathers'
+import { random } from 'lodash'
+
+import { identityProviderPath } from '@etherealengine/common/src/schemas/user/identity-provider.schema'
+import { userApiKeyPath, UserApiKeyType } from '@etherealengine/common/src/schemas/user/user-api-key.schema'
+import { InviteCode, UserName, userPath } from '@etherealengine/common/src/schemas/user/user.schema'
+import { Application } from '../../../declarations'
+import config from '../../appconfig'
+import { RedirectConfig } from '../../types/OauthStrategies'
+import getFreeInviteCode from '../../util/get-free-invite-code'
+import makeInitialAdmin from '../../util/make-initial-admin'
+import CustomOAuthStrategy, { CustomOAuthParams } from './custom-oauth'
+
+const logger = multiLogger.child({ component: 'engine:ecs:Apple' })
```

Comment:
Is this `logger` used in this file?

If so, I dont think we need to have `engine:ecs` in `component: 'engine:ecs:Apple'`

Rather it can be ``component: 'server-core:strategies:apple'`

Reply:
This was not needed, its been removed now.